### DISCUSSION TITLE
`agent-merging`: Allow customizing fields displayed in the dialog

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5721] [Feature Requests]
> In the merge record window, the default fields are added from the query?, I mean how can custom fileds be added to the merge récords window?

**Requested By:** Jordi at Barcelona

**Comment by maxpatiiuk** [comment]
Need more details on the use cases to know how to implement this.
Is this actually needed given that all conflicting fields are visible, and at any point you can uncheck "Show conflicting fields only" to see everything?
Or should there be a third option "Show non-empty fields only"?
Or should make sure to always display all the fields that are included in a query
